src/View.php

```diff
@@ -120,6 +122,16 @@ public function __construct(string $basePath, Theme $theme, EventDispatcherInter
         $this->logger = $logger;
     }
 
+    public function setPlaceholderSalt(string $salt): void
+    {
+        $this->placeholderSalt = $salt;
+    }
+
+    public function getPlaceholderSignature(): string
+    {
+        return md5($this->placeholderSalt);
```

Comment:
What about `sha1`?

Reply:
`md5` a little bit faster (~30%)

Reply:
ok

Reply:
The algorithm is almost irrelevant, so I changed it to CRC32, which is even faster.